pgstore.go

```diff
@@ -19,52 +20,47 @@ import (
 type PGStore struct {
 	Codecs  []securecookie.Codec
 	Options *sessions.Options
-	path    string
-	DbMap   *gorp.DbMap
+	Path    string
+	DbPool  *sql.DB
 }
 
-// Session type
-type Session struct {
-	Id         int64     `db:"id"`
-	Key        string    `db:"key"`
-	Data       string    `db:"data"`
-	CreatedOn  time.Time `db:"created_on"`
-	ModifiedOn time.Time `db:"modified_on"`
-	ExpiresOn  time.Time `db:"expires_on"`
+// PGSession type
+type PGSession struct {
+	ID         int64
```

Comment:
This is also a breaking change if I'm not mistaken. This is why we ignore the error in gometalinter.